login and register

LourenzBani opened the pull request.

Aspyse commented:
- Use responsive units (rem) in CSS.
- Do NOT run `npm init`. This will ruin our project setup and dependencies.
    - Revert your version of package.json and package-lock.json to the one in the main branch then run `npm install`.
- Please lint your code. I have already set this up in the original package.json so after reverting, you only have to run `node run standard`.

LourenzBani commented:
converted px to rem for login and register CSS.
reverted package.json and package-lock.json and run npm install.
used npm run standard (some variables were left unused though).
